README.md

```diff
@@ -65,6 +65,16 @@ def hello():
     return 'hello world'
 ```
 
+add key, value for your tracing record, like:
+
+```python
+
+zipkin.logging(id=1, user_id=2)
```

Comment:
@adriancole hmmm... logging maybe not the name. I think `logging` here means send the span with annotations.